adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -1111,7 +1734,65 @@ private[adam] class VariantContextConverter(
     }
   }
 
-  private[converters] def makeBdgGenotypeConverter(
+  def makeVariantExtractFn(
+    headerLines: Seq[VCFHeaderLine]): (ADAMVariantContext) => HtsjdkVariantContext = {
+
+    val attributeFns: Iterable[(Map[String, String]) => Option[(String, java.lang.Object)]] = headerLines
+      .flatMap(hl => hl match {
+        case il: VCFInfoHeaderLine => {
+
+          // get the id of this line
+          val key = il.getID
+
+          // filter out the lines that we already support
+          if (SupportedHeaderLines.infoHeaderLines
+            .find(_.getID == key)
+            .isEmpty) {
+
+            None
+          } else {
+            Some(extractorFromInfoLine(il))
+          }
+        }
+        case _ => None
+      })
+
+    def convert(vc: ADAMVariantContext): HtsjdkVariantContext = {
+      val v = vc.variant.variant
+      val builder = new VariantContextBuilder()
```

Comment:
I would add an inline doc above this call noting that VCF is 1-based closed intervals. This is obvious from the code, but I always like to list it inline to call attention to the coordinate format change.